aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/write_set_too_large.exp

```diff
@@ -0,0 +1,4 @@
+processed 3 tasks
+
+task 2 'run'. lines 29-29:
+Error: Failed to execute transaction. ExecutionStatus: status STORAGE_WRITE_LIMIT_REACHED of type Execution
```

Comment:
@wrwg and folks:

I can only get this result while running with `--release` because I hit this assertion failure with the debug build:
https://github.com/move-language/move/blob/ed592c619e87160eb226662581e268d11fbbb426/language/move-core/types/src/vm_status.rs#L134

Is it a (big) concern? What's the best fix?